src/logic/disk-game-logic.js

```diff
@@ -28,6 +28,12 @@ export default class DiskGameLogic {
     this._level = DEFAULT_LEVEL;
   }
 
+  end() {
+    let lights = this.store.data.get('lights');
+    lights.deactivateAll();
+    _.each(lights.stripIds, (id) => lights.setIntensity(id, null, 0));;
```

Comment:
Don't use lodash in this codebase please! :) Just use the regular forEach method on the array. Please make this change throughout